Doc/extending/extending.rst

```diff
@@ -204,24 +204,34 @@ usually declare a static object variable at the beginning of your file::
 
    static PyObject *SpamError;
```

Comment:
This is a problem if the module is loaded more than once (for example, in multiple subinterpreters).
In that case, `spam_exec` would be executed multiple times, each time calling `PyErr_NewException` and setting `SpamError` to the newly created error object.

This pointer needs to be part of per-module state.